### PR TITLE
add Invite ExpiresAt field

### PR DESCRIPTION
**Pull request by Aderlx**
As you can see. What do you think about adding a ExpiresAt field ?

### CODE REVIEW

File: structs.go [hunk 1]
```diff
@@ -246,6 +246,8 @@ type Invite struct {
 	// will only be filled when using InviteWithCounts
 	ApproximatePresenceCount int `json:"approximate_presence_count"`
 	ApproximateMemberCount   int `json:"approximate_member_count"`
+
+	ExpiresAt time.Time `json:"expires_at"`
```
Comment (FedorLap2006) [inline structs.go:250]: According to docs this field is optional, maybe better would be to make it a pointer?
```suggestion
	ExpiresAt *time.Time `json:"expires_at"`
```

**Review by FedorLap2006 — CHANGES_REQUESTED** [review]

File: structs.go (hunk 1 above)
Comment (Aderlx) [reply structs.go:250]: sure

**Review by Aderlx — COMMENTED** [review]
optional expires_at

**Review by FedorLap2006 — APPROVED** [review]

**Comment by FedorLap2006** [comment]
Thanks for your contribution!

**Pull request merged**